arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2130,11 +2120,35 @@
         },
         "backupManagementType": {
           "description": "Type of backup managmenent for the job. Possible value: AzureIaasVM.",
```

Comment:
Good with creating enums. 

Probably should trim off the `Possible value: AzureIaasVM.` text